### PR TITLE
New header logo design

### PR DESCRIPTION
**Pull request by LaszloNA**
Hi Ken, Siobhan, Ago,

Please review my code change on the new header logo and navigation bar design.

Thank you,
Laszlo

### **Changes made:**

1. Added the new NA logo from the "New header logo design" Trello card - NEW_NA_Logo.png
2. Added "United Kingdom" text to the new header with the same color as the NA logo
3. **Changed around style.css:**

- > added new navbar-menu class
- > changed navbar element
- > re-formatted the positioning of elements in the navbar
- > made the menu navbar "sticky-top"

_All changes made should be responsive._

### CODE REVIEW

**Comment by SiobhanCait** [comment]
Thank you Laszlo! I just checked the deployment on my iphone..
The sticky menu works great and the United Kingdom text behaves as expected, fonts look good too.  However, the one thing it that the logo doesn't quite 'shrinkenough' for this size. See screenshots. 
IMG_6791
IMG_6790

**Comment by LaszloNA** [comment]
Screenshot_20210207-225702_Chrome
The logo also falls into the right specs on mobile as well

File: app/templates/components/header.html
Comment (kenlt-uk) [inline app/templates/components/header.html:7]: I'm not sure that changing from `row` to `id` is doing anything. 
I think if you strip out the `id` name and its values it will still behave in the same way

File: app/static/css/style.css
Comment (kenlt-uk) [inline app/static/css/style.css:120]: This will not target anything in the html unless an `id` is set to `id`, so it should probably be removed as redundant.

**Review by kenlt-uk — CHANGES_REQUESTED** [review]
I don't think your code changes add anything to how it works, it is better to remove them to reduce confusion. It should work in the same way.

The reason why the tests are now passing is because the `row` class has been removed.

**Comment by SiobhanCait** [comment]
There is still the same issue with the sizing of the logo on iphone. If it is too much to solve now we can approve like this and come back to it? Or try to solve it now..? Let me know what makes sense from a technical point of view.

**Comment by LaszloNA** [comment]
In my simulation after this change the picture is scaling accordingly. I hope you will observe the same behaviour :)
Screenshot 2021-02-09 at 01 06 26

**Comment by LaszloNA** [comment]
I have changed the scaling on the logo in the header and should not fall out the window anymore regardless the viewport.

**Review by SiobhanCait — APPROVED** [review]
Checked new deployment on mobile, looks great!

**Review by kenlt-uk — APPROVED** [review]
LGTM, check the preview site once deployed and then let me know if its ok or not.

**Comment by kenlt-uk** [comment]
@LaszloNA feel free to merge, then wait about 10 minutes for it to be deployed to preview site

**Comment by LaszloNA** [comment]
I need your approval for merging =)

**Pull request merged**
